bumpversion/__init__.py

```diff
@@ -152,7 +152,9 @@ def tag(cls, name, message):
         command = ["git", "tag", name]
         if message:
             command += ['--message', message]
-        subprocess.check_output(command)
+        elif message is None:
+            command += ['--annotate']
+        subprocess.call(command)
```

Comment:
Turns out I was confused with check_output (which is 2.7+), check_call exists since 2.5 so even less of a problem.